docker-compose.yml

```diff
@@ -0,0 +1,63 @@
+version: "3.8"
+name: irma_email_issuer
+
+services:
+  # Irma issuer service
+  irmaserver:
+    image: ghcr.io/privacybydesign/irma:v0.13.2
+    container_name: irma_server
```

Comment:
Maybe you can remove the container names, because I got a name clash with the irma_server from the SMS issuer (even though that one wasn't running that was a problem).